tests/load_interrupt/dmctl.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/pingcap/dm/tests/utils"
+)
+
+func main() {
+	cli, err := utils.CreateDmCtl("127.0.0.1:8261")
+	if err != nil {
+		utils.ExitWithError(err)
+	}
+	conf := os.Args[1]
+	err = utils.StartTask(context.Background(), cli, conf, nil)
```

Comment:
DNM this PR until #112 merged and re-use dmctl binaries?

Reply:
not big problem, either this pr or #112 is merged first, the other one must fix to use the global dmctl binaries